### DISCUSSION TITLE
Have question trying to fix Ikea Fyrtyr Blind quirk

### DISCUSSION THREAD

**Discussion by alvinchen1** [zigpy/zha-device-handlers#2754] [Q&A]
My first attempt at improving a ZHA quirk. Very new, but wanted to try. Here's the original

https://github.com/zigpy/zha-device-handlers/blob/dev/zhaquirks/ikea/blinds.py

My version is https://github.com/alvinchen1/zha-device-handlers/commit/3c4c7f95f89359b133ed90fc10ea556054fc9c05

Some background, with firmware 0x24040011, IKEA introduced a new output cluster 3. (see https://github.com/zigpy/zha-device-handlers/issues/2418).  So following what was written before, I added a new class and made my changes there.  Seems to work, when I copy the file into custom_zha_quirks and reboot my HA, I see my devices showing my custom quirk where they did not show it before.  
<img width="243" alt="image" src="https://github.com/zigpy/zha-device-handlers/assets/26409535/04a3e8aa-7671-4730-b789-9c454a457930">

Seemed like things were fine, but then I removed a device that died, added a replacement one, let it update firmware to 0x24040011, and it now is showing as a Window_Covering device that supports both lift_percentage as well as tilt_percentage
<img width="439" alt="image" src="https://github.com/zigpy/zha-device-handlers/assets/26409535/6236e0bd-085b-4d13-b5ef-e638bd431457">
The lift controls work 100%, the tilt control does not do anything, and if you attempt to use it, will generate an error.

<img width="616" alt="image" src="https://github.com/zigpy/zha-device-handlers/assets/26409535/2907ec93-052b-4ee1-8c50-4f661bfa41db">

When I download diagnostics, I see:

          },
          "0x0102": {
            "endpoint_attribute": "window_covering",
            "attributes": {
              "0x0008": {
                "attribute_name": "current_position_lift_percentage",
                "value": 50
              }
            },
            "unsupported_attributes": {
              "0x0009": {
                "attribute_name": "current_position_tilt_percentage"
              }
            }
          },

What's strange is it is only on this new device, so to test, I removed another device, factory reset it, removed the battery for 1 minute, added it back in and now this one has tilt_percentage as well.  So my question is, I looked over other people's quirks trying to figure it out, but I can't seem to figure it out, but how do I write my quirk to IGNORE attribute 0x0009?  Also don't understand why the other classes in the file worked, but my version is somehow picking up attribute 0x0009.  Is this the right way to do this? 
 Basically everything else is working, but I need the quirk to not pay attention to 0x0102,attribute.0x0009.  Thoughts?
